zkvm/src/signature/multikey.rs

```diff
@@ -0,0 +1,58 @@
+use crate::errors::VMError;
+use crate::signature::VerificationKey;
+use crate::transcript::TranscriptProtocol;
+use curve25519_dalek::ristretto::RistrettoPoint;
+use curve25519_dalek::scalar::Scalar;
+use merlin::Transcript;
+
+#[derive(Clone)]
+pub struct Multikey {
+    transcript: Transcript,
+    X_agg: VerificationKey,
+}
+
+impl Multikey {
+    pub fn new(pubkeys: Vec<VerificationKey>) -> Result<Self, VMError> {
```

Comment:
maybe use "lightweight" Option instead of Result here? We do not have a lot of errors to throw via `?` shortcut.

Reply:
👍